src/components/PupilViews/PupilQuiz/PupilQuiz.view.tsx

```diff
@@ -28,6 +28,11 @@ import {
 import { useGetSectionLinkProps } from "@/components/PupilComponents/pupilUtils/lessonNavigation";
 import { QuizQuestionAnswers } from "@/node-lib/curriculum-api-2023/queries/pupilLesson/pupilLesson.schema";
 
+const OakTooltip = dynamic(
```

Comment:
Cool, but can you explain why this is necessary and how it caused the bug in the first place. Also is this something that should be done in Oak Components instead of here. If not there should be a comment explaining why turning off SSR for this component is necessary.